src/sentry/static/sentry/app/views/performance/transactionSummary/content.tsx

```diff
@@ -202,27 +201,24 @@ class SummaryContent extends React.Component<Props, State> {
               eventView={eventView}
               totalValues={totalCount}
             />
-            {totalValues === null ? (
-              <Placeholder height="352px" />
-            ) : (
-              <TransactionsList
-                location={location}
-                organization={organization}
-                eventView={eventView}
-                titles={[t('id'), t('user'), t('duration'), t('timestamp')]}
-                handleDropdownChange={this.handleTransactionsListSortChange}
-                generateLink={{
-                  id: generateTransactionLink(transactionName),
-                }}
-                baseline={transactionName}
-                handleBaselineClick={this.handleViewDetailsClick}
-                handleCellAction={this.handleCellAction}
-                handleOpenInDiscoverClick={this.handleDiscoverViewClick}
-                {...getTransactionsListSort(location, {
-                  p95: totalValues.p95,
-                })}
-              />
-            )}
+            <TransactionsList
+              location={location}
+              organization={organization}
+              eventView={eventView}
+              titles={[t('id'), t('user'), t('duration'), t('timestamp')]}
+              handleDropdownChange={this.handleTransactionsListSortChange}
+              generateLink={{
+                id: generateTransactionLink(transactionName),
+              }}
+              baseline={transactionName}
+              handleBaselineClick={this.handleViewDetailsClick}
+              handleCellAction={this.handleCellAction}
+              handleOpenInDiscoverClick={this.handleDiscoverViewClick}
+              {...getTransactionsListSort(location, {
+                p95: totalValues ? totalValues.p95 : 0,
```

Comment:
A little trick with optional chaining and the nullish coalescing operator. You can even chain a bunch of these together like `a?.b?.c?.d`.

```suggestion
                p95: totalValues?.p95 ?? 0,
```

Reply:
hm, i knew about `?.` but forgot about `??`. actually now that i think about it, `totalValues?.p95 || 0` would actually correct here but unnecessarily confusing